wdfn-server/waterdata/templates/wdfn.html

```diff
@@ -0,0 +1,173 @@
+{% extends 'base_wdfn.html' %}
```

Comment:
I really like how you have implemented this with components for the USWDS elements.